Automattic/newspack-theme
# chore: fix semantic-release dependencies

### All Submissions:






### Changes proposed in this Pull Request:



Removes redundant dependencies which may conflict with the ones in `newspack-scripts`.

### How to test the changes in this Pull Request:

1. Run `npm ci --legacy-peer-deps && npm run semantic-release` and confirm you see output like this:

```bash
➜  newspack-theme git:(epic/ras-acc) npm run semantic-release

> newspack@1.89.0-epic-ras-acc.1 semantic-release
> semantic-release

[12:44:39 PM] [semantic-release] › ℹ  Running semantic-release version 19.0.5
[12:44:39 PM] [semantic-release] › ✔  Loaded plugin "verifyConditions" from "@semantic-release/npm"
[12:44:39 PM] [semantic-release] › ✔  Loaded plugin "verifyConditions" from "@semantic-release/github"
[12:44:39 PM] [semantic-release] › ✔  Loaded plugin "analyzeCommits" from "@semantic-release/commit-analyzer"
[12:44:39 PM] [semantic-release] › ✔  Loaded plugin "generateNotes" from "@semantic-release/release-notes-generator"
[12:44:39 PM] [semantic-release] › ✔  Loaded plugin "prepare" from "@semantic-release/changelog"
[12:44:39 PM] [semantic-release] › ✔  Loaded plugin "prepare" from "@semantic-release/npm"
[12:44:39 PM] [semantic-release] › ✔  Loaded plugin "prepare" from "semantic-release-version-bump"
[12:44:39 PM] [semantic-release] › ✔  Loaded plugin "prepare" from "@semantic-release/git"
[12:44:39 PM] [semantic-release] › ✔  Loaded plugin "publish" from "@semantic-release/npm"
[12:44:39 PM] [semantic-release] › ✔  Loaded plugin "publish" from "@semantic-release/github"
[12:44:39 PM] [semantic-release] › ✔  Loaded plugin "addChannel" from "@semantic-release/npm"
[12:44:39 PM] [semantic-release] › ✔  Loaded plugin "addChannel" from "@semantic-release/github"
[12:44:39 PM] [semantic-release] › ✔  Loaded plugin "success" from "@semantic-release/github"
[12:44:39 PM] [semantic-release] › ✔  Loaded plugin "fail" from "@semantic-release/github"
[12:44:39 PM] [semantic-release] › ⚠  This run was not triggered in a known CI environment, running in dry-run mode.
```

### Other information:
